packages/drawer/src/Drawer.tsx

```diff
@@ -0,0 +1,86 @@
+import { CloseIcon } from '@littlespoon/icons'
+import React, { PropsWithChildren, useEffect, useState } from 'react'
+import { FocusOn } from 'react-focus-on'
+
+import Backdrop from './Backdrop'
+import { DrawerBase, DrawerCloseButton, DrawerContent } from './DrawerBase'
+import Portal from './Portal'
+
+export type DrawerProps = PropsWithChildren<{
+  /**
+   * Identifier of the element that will be used to label the Drawer.
+   */
+  ariaLabelledBy?: string
+
+  /**
+   * Close button title.
+   * @defaultValue `'Close'`
+   */
+  closeButtonTitle?: string
+
+  /**
+   * Close handler.
+   */
+  onClose?: () => void
+
+  /**
+   * If `true`, the component is shown.
+   * @defaultValue `false`
+   */
+  open?: boolean
+
+  /**
+   * If `true`, the close button will be shown.
+   * @defaultValue `false`
+   */
+  showCloseButton?: boolean
+}>
+
+export default function Drawer({
+  ariaLabelledBy,
+  children,
+  closeButtonTitle = 'Close',
+  onClose,
+  open = false,
+  showCloseButton = false,
+}: DrawerProps) {
+  const [isOpen, setIsOpen] = useState(false)
+
+  useEffect(() => {
+    setIsOpen(open)
+  }, [open])
+
+  if (!isOpen) {
+    return null
+  }
+
+  const handleClose = () => {
+    setIsOpen(false)
+
+    if (!onClose) {
+      return
+    }
+    onClose()
+  }
+
+  return (
+    <Portal>
+      <FocusOn onEscapeKey={handleClose}>
+        <Backdrop onClick={handleClose} open={isOpen} />
+        <DrawerBase aria-labelledby={ariaLabelledBy} aria-modal role="dialog">
+          {showCloseButton && (
+            <DrawerCloseButton
+              aria-label={closeButtonTitle}
+              onClick={handleClose}
+              title={closeButtonTitle}
+            >
+              {/* TODO: Remove <title> in CloseIcon */}
```

Comment:
Will this be done in current PR or future PR?

Reply:
I realized that the SVG content of the `CloseIcon` component includes a `<title>` and `<desc>` elements (see https://github.com/little-spoon-dev/design-system/blob/main/packages/icons/src/CloseIcon.tsx#L26-L27), then, when the close button in the `Drawer` component is hovered it displays `Close icon` in its tooltip:

<img width="90" alt="image" src="https://user-images.githubusercontent.com/369338/187740429-b193baa8-89cb-4e5b-b9d3-d3e00183586b.png">

I put this `TODO` comment in order to ask you what to do since of course, removing stated elements from the `CloseIcon` component SVG content is not part of this MR.

Reply:
I think we can keep it. I don't think this breaks accessibility, just something that is not ideal and not deal breaker